challenge-02/README.md

```diff
@@ -35,19 +40,25 @@ Crie uma função com as seguintes características:
     Preencha todos os valores corretamente!
 3. O retorno da função deve ser a multiplicação dos 3 argumentos, somando `2` ao resultado da multiplicação.
 */
-?
+function tresArg(a, b, c) {
+	if (a && b && c !== undefined) {
```

Comment:
-1
Aqui acho que a sua intenção era testar se `a`, `b` e `c` são diferentes de `undefined`, correto?
Se for isso, para fazer essa verificação, é necessário testar cada variável separadamente. Esse seu IF quer dizer:
_Se `a` estiver definido, e `b` estiver definido, e `c` for diferente de indefinido (`undefined`), então..._

Se você perceber, as 3 validação são basicamente a mesma! Nesse caso, você pode testar somente com:

```js
if( a && b && c ) {
```

Entende?